packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx

```diff
@@ -27,7 +27,7 @@ const CollapsibleTradeParams = ({
 }) => {
     React.useEffect(() => {
         if (previous_symbol && is_allow_equal && has_allow_equals) setIsTradeParamsExpanded(true);
-    }, [previous_symbol]);
+    }, [previous_symbol, is_allow_equal, has_allow_equals, setIsTradeParamsExpanded]);
```

Comment:
do we need to wrap `setIsTradeParamsExpanded` in a callback here?

Reply:
Ignore this one